rust/cubestore/cubestore/src/store/compaction.rs

```diff
@@ -827,11 +833,13 @@ async fn write_to_files_by_keys(
     Ok(row_counts)
 }
 
+///Makes `SendableRecordBatchStream` with result of merging  persistent chunk `l` and in-memory chunk `r`
 async fn merge_chunks(
```

Comment:
I'm going to attempt to help with the English, while leaving the payload up to you.

"/// Builds a `SendableRecordBatchStream` containing the result of merging a persistent chunk `l` with an in-memory chunk `r`.

Reply:
BTW, this is exactly the kind of 'brief docstring' I'm looking for, to give some orientation to the casual reader.